integration-cli/docker_cli_network_unix_test.go

```diff
@@ -227,6 +318,27 @@ func (s *DockerNetworkSuite) TestDockerNetworkIpamMultipleNetworks(c *check.C) {
 	}
 }
 
+func (s *DockerNetworkSuite) TestDockerNetworkCustomIpam(c *check.C) {
+	// Create a bridge network using custom ipam driver
+	out, _, err := dockerCmdWithError("network", "create", "--ipam-driver", dummyIpamDriver, "br0")
```

Comment:
I fell back to `dockerCmdWithError` because in case of failure, the error thrown during the integ cli run won't contain any valuable information about the failure.